packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -23,6 +23,9 @@ const testContext = vm.createContext({
 });
 testContext.global = testContext;
 
+const jest = {};
+jest.expect = global.expect;
```

Comment:
Not adding `expect` to the `testContext`?

It would also be really nice to get `describe`/`it` working inside the exec fixtures, but that would need a significant rework of how they are written.